app/javascript/components/users/account_activation/ConfirmTerms.jsx

```diff
@@ -0,0 +1,96 @@
+// BigBlueButton open source conferencing system - http://www.bigbluebutton.org/.
+//
+// Copyright (c) 2022 BigBlueButton Inc. and by respective authors (see below).
+//
+// This program is free software; you can redistribute it and/or modify it under the
+// terms of the GNU Lesser General Public License as published by the Free Software
+// Foundation; either version 3.0 of the License, or (at your option) any later
+// version.
+//
+// Greenlight is distributed in the hope that it will be useful, but WITHOUT ANY
+// WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+// PARTICULAR PURPOSE. See the GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License along
+// with Greenlight; if not, see <http://www.gnu.org/licenses/>.
+
+import React, { useState } from 'react';
+import {
+  Button, Card, Stack,
+} from 'react-bootstrap';
+import { useTranslation } from 'react-i18next';
+import Spinner from '../../shared_components/utilities/Spinner';
+import Logo from '../../shared_components/Logo';
+import { useAuth } from '../../../contexts/auth/AuthProvider';
+import useUpdateUser from '../../../hooks/mutations/users/useUpdateUser';
+
+export default function ConfirmTerms() {
+  const { t } = useTranslation();
+
+  const currentUser = useAuth();
+  const updateUserAPI = useUpdateUser(currentUser?.id);
+  const [isCheckedTerms, setIsCheckedTerms] = useState(currentUser?.confirm_terms || false);
+  const [isCheckedEmails, setIsCheckedEmails] = useState(currentUser?.email_notifs || false);
+
+  // Update the user's confirm_terms value when the button is clicked
+  const handleConfirmTerms = () => {
+    updateUserAPI.mutate({
+      confirm_terms: isCheckedTerms,
+      email_notifs: isCheckedEmails,
+    });
+  };
+
+  return (
+    <div className="vertical-center">
+      <div className="text-center pb-4">
+        <Logo />
+      </div>
+      <Card className="col-md-4 mx-auto p-4 border-0 card-shadow">
```

Comment:
We can increase the width of the box here abit to stop the title from wrapping around